src/sections/node/info.tsx

```diff
@@ -156,23 +160,47 @@ function InfoPage() {
         >
           <tbody>
             <tr>
-              <th>Eligible</th>
+              <Tooltip title="Whether or not your node is eligible to connect to the Monte Rosa network.">
+                <th>Eligible</th>
+              </Tooltip>
```

Comment:
All the tooltips should encapsulate the text and not the td or th
The table components shouldn't be separated if there is no need

```suggestion
             <th>
                  <Tooltip 
                       title="Whether or not your node is eligible to connect to the Monte Rosa network.">
                  Eligible
                  </Tooltip>
             </th>
```